README.md

```diff
@@ -0,0 +1,22 @@
+# World Portals
+
+A module to add teleportation portals. Portals currently occupy a block and transport any entities that enter the occupied block to the portals destination world position.
+
+## Future Features
+
+- Block-linked portals that remove when the block is removed.
+- Cross world portals
+
+## PortalComponent
+A portal component represents a one way portal and has a block coordinate location and a world position destination.
+
+## PortalRasterizer & related.
```

Comment:
"PortalRasterizer & related."
Change the & to and. Further, and remove the period at the end of the heading. Further, "& related." : related what? It ends way too abruptly.
Note that this heading should also become a subheading under `Implementation`.